wordfreq/__init__.py

```diff
@@ -165,17 +84,13 @@ def tokenize(text, lang):
     first, so that they can be expected to match the data.
     """
     if lang == 'ja':
-        global mecab_tokenize
-        if mecab_tokenize is None:
-            from wordfreq.mecab import mecab_tokenize
+        from wordfreq.mecab import mecab_tokenize
```

Comment:
...for what it's worth, even though I brough this up, I did just do the timing test, and it's not necessarily positive:

```
In [7]: %%timeit
   ...: global lumi_science
   ...: if lumi_science is None:
   ...:     import lumi_science
   ...: 
10000000 loops, best of 3: 39.8 ns per loop

In [8]: %%timeit
   ...: import lumi_science
   ...: 
1000000 loops, best of 3: 553 ns per loop
```

On the other hand, you could argue that 500 ns is tiny compared to the run time of `mecab_tokenize()` in any event.  (Is it?)